samples/matter/common/src/board/led_widget.cpp

```diff
@@ -11,7 +11,9 @@
 #endif
 #include <zephyr/kernel.h>
 
-static LEDWidget::LEDWidgetStateUpdateHandler sStateUpdateCallback;
+static Nordic::LEDWidget::LEDWidgetStateUpdateHandler sStateUpdateCallback;
```

Comment:
Consider moving such global definitions inside `Nordic` namespace to shorten the type.